dev/tests/static/testsuite/Magento/Test/Integrity/Magento/Framework/Cache/ConfigTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types = 1);
+
+namespace Magento\Test\Integrity\Magento\Framework\Cache;
+
+/**
+ * Unit test of the cache configuration
+ */
+class ConfigTest extends \PHPUnit\Framework\TestCase
```

Comment:
You can definately import the classes :-)

Reply:
Hello @lbajsarowicz 
Thank you for your remark.
I have fixed it. Please check.
Thank you in advance.